### DISCUSSION TITLE
CKolkey Appreciation Post

### DISCUSSION THREAD

**Discussion by wrightjjw** [NeogitOrg/neogit#576] [General]
I just wanted to thank @CKolkey not just for the massive improvements that have just been merged in, but also for the very quick patches for some issues that were brought forth today!

This is such a fantastic plugin and it's great to see so much passion and initiative poured into an open source project like this! Thank you to all the contributors for helping make Neogit what it is today. I'm excited to see where the plugin goes from here!

**Comment by CKolkey** [comment]
You're too kind - I'm really glad this all got to see the light of day, and that people think its useful 🥳
